docs/src/modules/components/AppTableOfContents.js

```diff
@@ -204,6 +217,42 @@ export default function AppTableOfContents(props) {
     <Nav aria-label={t('pageTOC')}>
       <NoSsr>
         <TableOfContentsBanner />
+        {showAddJob && (
+          <Link
+            href="/careers/"
+            underline="none"
+            sx={{
+              color: 'text.secondary',
+              overflow: 'hidden',
+              p: 1,
+              display: 'block',
+              '& img': {
+                display: 'block',
+              },
+              border: '1px solid',
+              borderRadius: 1,
+              borderColor:
+                theme.palette.mode === 'dark'
+                  ? theme.palette.primaryDark[700]
+                  : theme.palette.grey[200],
+              fontSize: '0.9rem',
+            }}
+          >
+            <img
+              src={`/static/hiring-toc-${theme.palette.mode}.png`}
```

Comment:
Oops, I forgot to remove these images from /docs/public/static/, it's legacy, it's not using our older brand design

```suggestion
```

Reply:
For the new location of the new image, I think that we should start to have namespaces for the static images (CDN), e.g. https://mui.com/static/x/, https://mui.com/static/toolpad/, etc. I guess `/docs/public/static/docs/` that would then be using https://mui.com/static/docs/ would make sense here? This image is for our docs infrastructure. 

For the new image, you might look at https://www.figma.com/file/vOx7rzFQZ9e54W81NReDdU/Assets?node-id=718%3A26, unless Danilo has a new version.